### PR TITLE
Add test case for DashboardUserMapper and fix bugs

### PR DESCRIPTION
**Pull request by peiht**
Make sure that:

- [ yes ] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [ yes ] You submit test cases (unit or integration tests) that back your changes.
- [ yes ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+import static org.hamcrest.Matchers.greaterThan;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+/**
+ * test case for DashboardUserMapper.
+ *
+ * @author peiht
+ */
+public class DashboardUserMapperTest extends AbstractSpringIntegrationTest {
```
Comment (yu199195) [inline soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java:43]: add final

File: soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        int count = dashboardUserMapper.insertSelective(record);
+        assertThat(count, comparesEqualTo(1));
+    }
+
+    @Test
+    public void testSelectById() {
+        DashboardUserDO record = buildDashboardUserDO();
+        int count = dashboardUserMapper.insert(record);
+        assertThat(count, comparesEqualTo(1));
+
```
Comment (yu199195) [inline soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java:67]: remove empty line

File: soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        assertNotNull(result);
+    }
+
+    @Test
+    public void testFindByQuery() {
+        DashboardUserDO record = buildDashboardUserDO();
+        record.setUserName("adminSoul");
+        int count = dashboardUserMapper.insert(record);
+        assertThat(count, comparesEqualTo(1));
+
```
Comment (yu199195) [inline soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java:78]: remove empty line

File: soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        DashboardUserDO result = dashboardUserMapper.findByQuery(record.getUserName(), record.getPassword());
+        assertNotNull(result);
+    }
+
+    @Test
+    public void testSelectByQuery() {
+        DashboardUserDO record = buildDashboardUserDO();
+        int count = dashboardUserMapper.insert(record);
+        assertThat(count, comparesEqualTo(1));
+
```
Comment (yu199195) [inline soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java:88]: remove empty line

File: soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+        List<DashboardUserDO> result = dashboardUserMapper.selectByQuery(query);
+        assertThat(result.size(), greaterThan(0));
+    }
+
+    @Test
+    public void testCountByQuery() {
+        DashboardUserDO record = buildDashboardUserDO();
+        int count = dashboardUserMapper.insert(record);
+        assertThat(count, comparesEqualTo(1));
+
```
Comment (yu199195) [inline soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java:102]: remove empty line

File: soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+        int result = dashboardUserMapper.countByQuery(query);
+        assertThat(result, greaterThan(0));
+    }
+
+    @Test
+    public void testUpdate() {
+        DashboardUserDO record = buildDashboardUserDO();
+        int count = dashboardUserMapper.insert(record);
+        assertThat(count, comparesEqualTo(1));
+
```
Comment (yu199195) [inline soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java:114]: remove empty line

File: soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+        int result = dashboardUserMapper.update(record);
+        assertThat(result, comparesEqualTo(1));
+    }
+
+    @Test
+    public void testUpdateSelective() {
+        DashboardUserDO record = buildDashboardUserDO();
+        int count = dashboardUserMapper.insert(record);
+        assertThat(count, comparesEqualTo(1));
+
```
Comment (yu199195) [inline soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java:125]: remove empty line

File: soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+        int result = dashboardUserMapper.updateSelective(record);
+        assertThat(result, comparesEqualTo(1));
+    }
+
+    @Test
+    public void testDelete() {
+        DashboardUserDO record = buildDashboardUserDO();
+        int count = dashboardUserMapper.insert(record);
+        assertThat(count, comparesEqualTo(1));
+
```
Comment (yu199195) [inline soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java:136]: remove empty line

**Review by yu199195 — APPROVED** [review]

File: soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java (hunk 1 above)
Comment (peiht) [reply soul-admin/src/test/java/org/dromara/soul/admin/mapper/DashboardUserMapperTest.java:43]: OK, I will fix this

**Review by dengliming — APPROVED** [review]

**Pull request merged**